generators/client/templates/angular/src/main/webapp/app/_polyfills.ts

```diff
@@ -1,3 +1,27 @@
 /* tslint:disable */
+// Polyfills
 import 'reflect-metadata/Reflect';
+// import 'core-js/es6';
+// Added parts of es6 which are necessary for your project or your browser support requirements.
+import 'core-js/es6/symbol';
```

Comment:
This would bundle this for all browsers which os unnecessary overhead. May be we should add the polyfill to index.html directly using the IE conditional directive